### DISCUSSION TITLE
User account passthrough

### DISCUSSION THREAD

**Discussion by r-callow** [prasathmani/tinyfilemanager#483] [Q&A] [answered]
HI, is there a way the user account details can be passed to the php script?

e.g.  tinyfilemanager.php user='user' pass='1234'

**Comment by prasathmani** [comment, marked as answer]
yes, you can customize  the login option.
change the line no `276`.
```php
isset($_POST['fm_usr'], $_POST['fm_pwd'])
```
```php
isset($_GET['fm_usr'], $_GET['fm_pwd']))
```
pass `tinyfilemanager.php?fm_usr=user&fm_pwd=1234`

**Reply by hakkku** [reply]
> yes, you can customize the login option. change the line no `276`.
> 
> ```
> isset($_POST['fm_usr'], $_POST['fm_pwd'])
> ```
> 
> ```
> isset($_GET['fm_usr'], $_GET['fm_pwd']))
> ```
> 
> pass `tinyfilemanager.php?fm_usr=user&fm_pwd=1234`

Is this still viable? If not, how could I acomplish something like this?
